queries.sql

```diff
@@ -10,3 +11,77 @@ SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
 
+/* Tuesday's queries and transactions */
+
+-- TRANSACTION #1
+BEGIN;
+
+-- set set species column to unspecified
+UPDATE animals
+SET species = 'unspecified';
+
+-- roll back the transaction
+ROLLBACK;
+
```

Comment:
- [ ] Kindly make sure to verify that change was made before ROLLBACK, and also verify that change was undone after ROLLBACK by adding this (before and after the rollback:
` SELECT species from animals;`

Reply:
@sja-thedude Thank you for pointing this out, however, I had already added these verification commands in the terminal, which are visible in the attached screenshots above :arrow_up: